posthog/management/commands/migrate_kafka_data.py

```diff
@@ -0,0 +1,151 @@
+# This management command is intended to be used to migrate data from one Kafka
+# cluster to another. It is intended for the use case where the Kafka cluster
+# has suffered some catastrophic failure that causes us to migrate to a new
+# cluster. The old cluster may well come back up, and in which case we would
+# want to ensure that if there is any data in the old cluster that wasn't
+# consumed by the PostHog app when it was pointing at it, then that data is
+# transferred to the new cluster.
+#
+# We do not make any attempt at validation, so it's important to ensure that you
+# move data to and from the corresponding topics.
+#
+# We also do not validate that you use the correct consumer group ID, so it's
+# important that you use the same consumer group ID that the PostHog app is
+# using when consuming from the old cluster, otherwise you risk migrating data
+# that has already been consumed.
+#
+# By default the target Kafka cluster is the currently configured cluster in
+# Django settings.
+
+from django.core.management.base import BaseCommand
+from kafka import KafkaAdminClient, KafkaConsumer, KafkaProducer
+from kafka.errors import KafkaError
+from kafka.structs import TopicPartition
+
+from django.conf import settings
+
+
+class Command(BaseCommand):
+    help = "Migrate data from one Kafka cluster to another"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--from-topic",
+            required=True,
+            help="The topic to migrate data from",
+        )
+        parser.add_argument(
+            "--to-topic",
+            required=True,
+            help="The topic to migrate data to",
+        )
+        parser.add_argument(
+            "--from-cluster",
+            default=settings.KAFKA_URL,
+            help="The Kafka cluster to migrate data from",
+        )
+        parser.add_argument(
+            "--to-cluster",
+            default=settings.KAFKA_URL,
+            help="The Kafka cluster to migrate data to",
+        )
+        parser.add_argument(
+            "--consumer-group-id",
```

Comment:
```suggestion
            "--consumer-group-id",
            required=True,
```

We check it afterwards, but let's make the intent explicit here